### PR TITLE
Reorder terratest Workflow

### PR DESCRIPTION
**Pull request by k0da**
Right now workflow creates GSLB awaits for Ingress to come up, only then
deploys test app.

This represents corner case, while common case is to have and app to be
deployed at the time exposure object is supposed to be created.

This commit reorders Workflow in a way, podinfo deployed first, gslb
created afterwards.

Signed-off-by: Dinar Valeev <dinar.valeev@absa.africa>

### CODE REVIEW

File: terratest/utils/extensions.go [hunk 1]
```diff
@@ -211,6 +196,22 @@ func (w *Workflow) Start() (*Instance, error) {
...
+		w.t.Logf("Create ingress %s from %s", w.state.gslb.name, w.settings.gslbResourcePath)
+		k8s.KubectlApply(w.t, w.k8sOptions, w.settings.gslbResourcePath)
+		k8s.WaitUntilIngressAvailable(w.t, w.k8sOptions, w.state.gslb.name, 60, 1*time.Second)
+		ingress := k8s.GetIngress(w.t, w.k8sOptions, w.state.gslb.name)
+		require.Equal(w.t, ingress.Name, w.state.gslb.name)
+		w.settings.ingressName = w.state.gslb.name
+	}
+
+	// ingress
+	if w.settings.ingressResourcePath != "" {
```
Comment (somaritane) [inline terratest/utils/extensions.go:211]: is this intentional "NOOP" or am I missing something?

Comment (k0da) [reply terratest/utils/extensions.go:211]: Dunno, it was there so I'm leaving it as it is. Maybe @kuritka have an idea?

Comment (somaritane) [reply terratest/utils/extensions.go:211]: Yep, good to check. To me, this looks like a dead code.
Also surprised it got skipped the linters somehow, staticcheck would definitely treat this as an empty branch:
https://staticcheck.io/docs/checks/#SA9003

**Review by somaritane — APPROVED** [review]
@k0da LGTM

**Review by kuritka — APPROVED** [review]
